homework-3/src/components/Forms/FormSignIn.js

```diff
@@ -1,16 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Formik } from 'formik';
-import TitleForm from '../common/Titles/StylTitleForm';
-import FormInput from '../FormInput';
-import FormSign from '../common/partsOfTheForm/StylFormSign';
-import BtnSubmitFormSignIn from './../BtnsComponents/BtnSubmitFormSignIn';
+import { StylTitleForm } from '../common/Titles/';
+import { StylFormSign } from '../common/partsOfTheForm/';
+import { BtnSubmitFormSignIn } from './../BtnsComponents/';
+import { FormInput, LinkPage } from '../';
+import { routes } from './../../Router';
 import emailImg from './../../img/icons/icon-email.svg';
 import passwordlImg from './../../img/icons/icon-lock.svg';
-import LinkPage from '../LinkPage';
 
 const FormSignUp = () => {
-  let [isEmail, setEmail] = React.useState(false);
-  let [isPassword, setPassword] = React.useState(false);
+  const [isEmail, setEmail] = useState(false),
+    [isPassword, setPassword] = useState(false),
+    regulPassword = /(?=.*[A-Z]+)(?=.*[0-9])(?=.*[a-z]){6,}/,
+    regulEmail =
```

Comment:
As i told you before, move this into constats.ts, and import into every component, as the constat.

Reply:
Answer above